charts/kuberpult/templates/frontend-service.yaml

```
@@ -165,6 +165,8 @@ spec:
 {{- else }}
           value: ""
 {{- end }}
+        - name: KUBERPULT_WAIT_TIME_LIMIT
```

```suggestion
        - name: KUBERPULT_MAX_WAIT_DURATION_ROLLOUT
```
But I'm also ok keeping it as it is